### PR TITLE
Adds url to response

### PR DESCRIPTION
**Pull request by NejcZdovc**
Resolves https://github.com/brave/brave-browser/issues/9764

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Test Plan:
- not needed

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/payout_tokens.cc [hunk 1]
```diff
@@ -87,23 +87,19 @@ void PayoutTokens::RedeemPaymentTokens() {
...
   confirmations_client_->LoadURL(url, headers, body, content_type, method,
       callback);
 }
 
-void PayoutTokens::OnRedeemPaymentTokens(
-    const std::string& url,
-    const int response_status_code,
-    const std::string& response,
-    const std::map<std::string, std::string>& headers) {
+void PayoutTokens::OnRedeemPaymentTokens(const ledger::UrlResponse& response) {
```
[the remark this answers is not in the document]
Comment (tmancey) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/payout_tokens.cc:97]: also can you please add 'using UrlResponse = ledger::UrlResponse` in `confirmations_client.h` and then use `UrlResponse` for confirmations.

Could you please use `url_response` as the parameter name for confirmations

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/payout_tokens.h [hunk 2]
```diff
@@ -42,11 +42,7 @@ class PayoutTokens {
   Timer timer_;
 
   void RedeemPaymentTokens();
-  void OnRedeemPaymentTokens(
-      const std::string& url,
-      const int response_status_code,
-      const std::string& response,
-      const std::map<std::string, std::string>& headers);
+  void OnRedeemPaymentTokens(const ledger::UrlResponse& response);
```
Comment (tmancey) [inline vendor/bat-native-confirmations/src/bat/confirmations/internal/payout_tokens.h:45]: ditto

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.cc [hunk 3]
```diff
@@ -90,32 +90,29 @@ void RefillTokens::RequestSignedTokens() {
...
   confirmations_client_->LoadURL(url, headers, body, content_type, method,
       callback);
 }
 
-void RefillTokens::OnRequestSignedTokens(
-    const std::string& url,
-    const int response_status_code,
-    const std::string& response,
-    const std::map<std::string, std::string>& headers) {
+void RefillTokens::OnRequestSignedTokens(const ledger::UrlResponse& response) {
```
Comment (tmancey) [inline vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.cc:100]: ditto, with parameter on next line

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.cc [hunk 4]
```diff
@@ -143,31 +140,28 @@ void RefillTokens::GetSignedTokens() {
   auto method = request.GetMethod();
 
   auto callback = std::bind(&RefillTokens::OnGetSignedTokens,
-      this, url, _1, _2, _3);
+      this, _1);
```
Comment (tmancey) [inline vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.cc:143]: end of previous line

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.cc [hunk 5]
```diff
@@ -143,31 +140,28 @@ void RefillTokens::GetSignedTokens() {
...
   BLOG(5, UrlRequestToString(url, {}, "", "", method));
   confirmations_client_->LoadURL(url, {}, "", "", method, callback);
 }
 
-void RefillTokens::OnGetSignedTokens(
-    const std::string& url,
-    const int response_status_code,
-    const std::string& response,
-    const std::map<std::string, std::string>& headers) {
+void RefillTokens::OnGetSignedTokens(const ledger::UrlResponse& response) {
```
Comment (tmancey) [inline vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.cc:149]: ditto

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.h [hunk 6]
```diff
@@ -47,18 +47,10 @@ class RefillTokens {
   std::vector<BlindedToken> blinded_tokens_;
 
   void RequestSignedTokens();
-  void OnRequestSignedTokens(
-      const std::string& url,
-      const int response_status_code,
-      const std::string& response,
-      const std::map<std::string, std::string>& headers);
+  void OnRequestSignedTokens(const ledger::UrlResponse& response);
```
Comment (tmancey) [inline vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.h:50]: ditto

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.h [hunk 7]
```diff
@@ -47,18 +47,10 @@ class RefillTokens {
...
 
   void GetSignedTokens();
-  void OnGetSignedTokens(
-      const std::string& url,
-      const int response_status_code,
-      const std::string& response,
-      const std::map<std::string, std::string>& headers);
+  void OnGetSignedTokens(const ledger::UrlResponse& response);
```
Comment (tmancey) [inline vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.h:53]: ditto

File: vendor/bat-native-ledger/include/bat/ledger/ledger_client.h [hunk 8]
```diff
@@ -25,8 +25,7 @@ using PublisherInfoListCallback =
 using GetNicewareListCallback =
     std::function<void(const Result, const std::string&)>;
 using FetchIconCallback = std::function<void(bool, const std::string&)>;
-using LoadURLCallback = std::function<void(const int, const std::string&,
-    const std::map<std::string, std::string>& headers)>;
+using LoadURLCallback = std::function<void(const ledger::UrlResponse&)>;
```
Comment (tmancey) [inline vendor/bat-native-ledger/include/bat/ledger/ledger_client.h:28]: Ditto with regards to `using`, see above

File: vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger.mojom [hunk 9]
```diff
@@ -530,3 +530,10 @@ struct SKUTransaction {
   uint64 created_at;
 };
 
+struct UrlResponse {
+  string url;
+  int32 code;
```
Comment (tmancey) [inline vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger.mojom:535]: `status_code`

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/logging_util.cc [hunk 10]
```diff
@@ -67,16 +68,12 @@ std::string HeadersToString(
...
-    const std::string& response,
-    const std::map<std::string, std::string>& headers) {
-  const std::string formatted_headers = HeadersToString(headers);
+std::string UrlResponseToString(const ledger::UrlResponse& response) {
+  const std::string formatted_headers = HeadersToString(response.headers);
 
   return base::StringPrintf("URL Response:\n  URL: %s\n  Response "
-      "Status Code: %d\n  Response: %s\n  Headers:\n%s", url.c_str(),
-          response_status_code, response.c_str(), formatted_headers.c_str());
+      "Status Code: %d\n  Response: %s\n  Headers:\n%s", response.url.c_str(),
```
Comment (tmancey) [inline vendor/bat-native-confirmations/src/bat/confirmations/internal/logging_util.cc:75]: Please change `Response:` to `Body:` to match variable name

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/logging_util.h [hunk 11]
```diff
@@ -21,11 +21,7 @@ std::string UrlRequestToString(
     const std::string& content_type,
     const URLRequestMethod method);
 
-std::string UrlResponseToString(
-    const std::string& url,
-    const int response_status_code,
-    const std::string& response,
-    const std::map<std::string, std::string>& headers);
+std::string UrlResponseToString(const ledger::UrlResponse& response);
```
Comment (tmancey) [inline vendor/bat-native-confirmations/src/bat/confirmations/internal/logging_util.h:24]: param next line

File: components/brave_rewards/browser/rewards_service_impl.cc [hunk 12]
```diff
@@ -1087,13 +1103,25 @@ void RewardsServiceImpl::OnURLLoaderComplete(
     network::SimpleURLLoader* loader,
     ledger::LoadURLCallback callback,
     std::unique_ptr<std::string> response_body) {
+  if (!Connected()) {
+    return;
+  }
+
   DCHECK(url_loaders_.find(loader) != url_loaders_.end());
```
[the remark this answers is not in the document]
Comment (NejcZdovc) [reply components/brave_rewards/browser/rewards_service_impl.cc:1110]: fixed

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/ads_rewards.cc [hunk 13]
```diff
@@ -109,29 +109,25 @@ void AdsRewards::GetPaymentBalance() {
   auto content_type = request.GetContentType();
 
   auto callback = std::bind(&AdsRewards::OnGetPaymentBalance,
-      this, url, _1, _2, _3);
+      this, _1);
```
[the remark this answers is not in the document]
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/ads_rewards.cc:112]: fixed

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/ads_rewards.cc [hunk 14]
```diff
@@ -109,29 +109,25 @@ void AdsRewards::GetPaymentBalance() {
...
   BLOG(5, UrlRequestToString(url, headers, "", "", method));
   confirmations_client_->LoadURL(url, headers, "", "", method, callback);
 }
 
-void AdsRewards::OnGetPaymentBalance(
-    const std::string& url,
-    const int response_status_code,
-    const std::string& response,
-    const std::map<std::string, std::string>& headers) {
+void AdsRewards::OnGetPaymentBalance(const ledger::UrlResponse& response) {
```
[the remark this answers is not in the document]
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/ads_rewards.cc:118]: done

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/ads_rewards.cc [hunk 15]
```diff
@@ -147,37 +143,33 @@ void AdsRewards::GetAdGrants() {
...
   BLOG(5, UrlRequestToString(url, {}, "", "", method));
   confirmations_client_->LoadURL(url, {}, "", "", method, callback);
 }
 
-void AdsRewards::OnGetAdGrants(
-    const std::string& url,
-    const int response_status_code,
-    const std::string& response,
-    const std::map<std::string, std::string>& headers) {
+void AdsRewards::OnGetAdGrants(const ledger::UrlResponse& response) {
```
[the remark this answers is not in the document]
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/ads_rewards.cc:152]: done

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/ads_rewards.h [hunk 16]
```diff
@@ -39,18 +39,10 @@ class AdsRewards {
   WalletInfo wallet_info_;
 
   void GetPaymentBalance();
-  void OnGetPaymentBalance(
-      const std::string& url,
-      const int response_status_code,
-      const std::string& response,
-      const std::map<std::string, std::string>& headers);
+  void OnGetPaymentBalance(const ledger::UrlResponse& response);
```
[the remark this answers is not in the document]
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/ads_rewards.h:42]: done

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/ads_rewards.h [hunk 17]
```diff
@@ -39,18 +39,10 @@ class AdsRewards {
...
 
   void GetAdGrants();
-  void OnGetAdGrants(
-      const std::string& url,
-      const int response_status_code,
-      const std::string& response,
-      const std::map<std::string, std::string>& headers);
+  void OnGetAdGrants(const ledger::UrlResponse& response);
```
[the remark this answers is not in the document]
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/ads_rewards.h:45]: done

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/logging_util.cc [hunk 18]
```diff
@@ -67,16 +68,12 @@ std::string HeadersToString(
   return base::JoinString(formatted_headers, "\n");
 }
 
-std::string UrlResponseToString(
-    const std::string& url,
-    const int response_status_code,
-    const std::string& response,
-    const std::map<std::string, std::string>& headers) {
-  const std::string formatted_headers = HeadersToString(headers);
+std::string UrlResponseToString(const ledger::UrlResponse& response) {
```
[the remark this answers is not in the document]
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/logging_util.cc:71]: done

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/payout_tokens.cc [hunk 19]
```diff
@@ -87,23 +87,19 @@ void PayoutTokens::RedeemPaymentTokens() {
   auto content_type = request.GetContentType();
 
   auto callback = std::bind(&PayoutTokens::OnRedeemPaymentTokens,
-      this, url, _1, _2, _3);
+      this, _1);
```
[the remark this answers is not in the document]
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/payout_tokens.cc:90]: done

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/payout_tokens.cc (hunk 1 above)
[the remark this answers is not in the document]
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/payout_tokens.cc:97]: done

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.cc (hunk 3 above)
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.cc:100]: done

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.cc (hunk 4 above)
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.cc:143]: done

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.cc (hunk 5 above)
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.cc:149]: done

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.h (hunk 6 above)
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.h:50]: done

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.h (hunk 7 above)
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/refill_tokens.h:53]: done

File: vendor/bat-native-ledger/include/bat/ledger/ledger_client.h (hunk 8 above)
Comment (NejcZdovc) [reply vendor/bat-native-ledger/include/bat/ledger/ledger_client.h:28]: this looks good to me

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/logging_util.cc (hunk 10 above)
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/logging_util.cc:75]: done

File: vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger.mojom (hunk 9 above)
Comment (NejcZdovc) [reply vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger.mojom:535]: done

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/logging_util.h (hunk 11 above)
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/logging_util.h:24]: done

File: vendor/bat-native-ledger/include/bat/ledger/ledger_client.h (hunk 8 above)
Comment (tmancey) [reply vendor/bat-native-ledger/include/bat/ledger/ledger_client.h:28]: Confirmations follows the pattern `using UrlResponse = ledger::UrlResponse` in `confirmations_client.h` to remove `ledger` name space from confirmations code

Comment (NejcZdovc) [reply vendor/bat-native-ledger/include/bat/ledger/ledger_client.h:28]: but that is native ledger code not confirmation

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/logging_util.cc [hunk 20]
```diff
@@ -68,15 +69,13 @@ std::string HeadersToString(
...
-  const std::string formatted_headers = HeadersToString(headers);
+    const UrlResponse& url_response) {
+  const std::string formatted_headers = HeadersToString(url_response.headers);
 
   return base::StringPrintf("URL Response:\n  URL: %s\n  Response "
-      "Status Code: %d\n  Response: %s\n  Headers:\n%s", url.c_str(),
-          response_status_code, response.c_str(), formatted_headers.c_str());
+      "Status Code: %d\n  Body: %s\n  Headers:\n%s",
+          url_response.url.c_str(), url_response.status_code,
+             url_response.body.c_str(), formatted_headers.c_str());
```
Comment (tmancey) [inline vendor/bat-native-confirmations/src/bat/confirmations/internal/logging_util.cc:78]: indentation is wrong, needs 1 more space

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/redeem_token_unittest.cc [hunk 21]
```diff
@@ -194,18 +196,20 @@ TEST_F(ConfirmationsRedeemTokenTest,
...
         if (endpoint == fetch_payment_token_endpoint) {
           response_status_code = net::HTTP_OK;
-          response = R"({"id":"9fd71bc4-1b8e-4c1e-8ddc-443193a09f91","createdAt":"2020-04-20T10:27:11.717Z","type":"view","modifiedAt":"2020-04-20T10:27:11.736Z","creativeInstanceId":"70829d71-ce2e-4483-a4c0-e1e2bee96520","paymentToken":{"publicKey":"bPE1QE65mkIgytffeu7STOfly+x10BXCGuk5pVlOHQU=","batchProof":"FWTZ5fOYITYlMWMYaxg254QWs+Pmd0dHzoor0mzIlQ8tWHagc7jm7UVJykqIo+ZSM+iK29mPuWJxPHpG4HypBw==","signedTokens":["DHe4S37Cn1WaTbCC+ytiNTB2s5H0vcLzVcRgzRoO3lU="]}})";
+          body = R"({"id":"9fd71bc4-1b8e-4c1e-8ddc-443193a09f91","createdAt":"2020-04-20T10:27:11.717Z","type":"view","modifiedAt":"2020-04-20T10:27:11.736Z","creativeInstanceId":"70829d71-ce2e-4483-a4c0-e1e2bee96520","paymentToken":{"publicKey":"bPE1QE65mkIgytffeu7STOfly+x10BXCGuk5pVlOHQU=","batchProof":"FWTZ5fOYITYlMWMYaxg254QWs+Pmd0dHzoor0mzIlQ8tWHagc7jm7UVJykqIo+ZSM+iK29mPuWJxPHpG4HypBw==","signedTokens":["DHe4S37Cn1WaTbCC+ytiNTB2s5H0vcLzVcRgzRoO3lU="]}})";
         }
 
-        callback(response_status_code, response, response_headers);
+        ledger::UrlResponse response;
```
Comment (tmancey) [inline vendor/bat-native-confirmations/src/bat/confirmations/internal/redeem_token_unittest.cc:209]: UrlResponse

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/redeem_token_unittest.cc [hunk 22]
```diff
@@ -141,22 +141,24 @@ TEST_F(ConfirmationsRedeemTokenTest,
...
         } else if (endpoint == fetch_payment_token_endpoint) {
           response_status_code = net::HTTP_OK;
-          response = R"({"id":"9fd71bc4-1b8e-4c1e-8ddc-443193a09f91","createdAt":"2020-04-20T10:27:11.717Z","type":"view","modifiedAt":"2020-04-20T10:27:11.736Z","creativeInstanceId":"70829d71-ce2e-4483-a4c0-e1e2bee96520","paymentToken":{"publicKey":"bPE1QE65mkIgytffeu7STOfly+x10BXCGuk5pVlOHQU=","batchProof":"FWTZ5fOYITYlMWMYaxg254QWs+Pmd0dHzoor0mzIlQ8tWHagc7jm7UVJykqIo+ZSM+iK29mPuWJxPHpG4HypBw==","signedTokens":["DHe4S37Cn1WaTbCC+ytiNTB2s5H0vcLzVcRgzRoO3lU="]}})";
+          body = R"({"id":"9fd71bc4-1b8e-4c1e-8ddc-443193a09f91","createdAt":"2020-04-20T10:27:11.717Z","type":"view","modifiedAt":"2020-04-20T10:27:11.736Z","creativeInstanceId":"70829d71-ce2e-4483-a4c0-e1e2bee96520","paymentToken":{"publicKey":"bPE1QE65mkIgytffeu7STOfly+x10BXCGuk5pVlOHQU=","batchProof":"FWTZ5fOYITYlMWMYaxg254QWs+Pmd0dHzoor0mzIlQ8tWHagc7jm7UVJykqIo+ZSM+iK29mPuWJxPHpG4HypBw==","signedTokens":["DHe4S37Cn1WaTbCC+ytiNTB2s5H0vcLzVcRgzRoO3lU="]}})";
         }
 
-        callback(response_status_code, response, response_headers);
+        ledger::UrlResponse response;
```
Comment (tmancey) [inline vendor/bat-native-confirmations/src/bat/confirmations/internal/redeem_token_unittest.cc:158]: UrlResponse

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/redeem_token_unittest.cc [hunk 23]
```diff
@@ -256,7 +258,9 @@ TEST_F(ConfirmationsRedeemTokenTest,
           response_status_code = net::HTTP_NOT_FOUND;
         }
 
-        callback(response_status_code, response, response_headers);
+        ledger::UrlResponse response;
```
Comment (tmancey) [inline vendor/bat-native-confirmations/src/bat/confirmations/internal/redeem_token_unittest.cc:261]: UrlResponse

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/redeem_token_unittest.cc [hunk 24]
```diff
@@ -307,7 +309,9 @@ TEST_F(ConfirmationsRedeemTokenTest,
           response_status_code = net::HTTP_INTERNAL_SERVER_ERROR;
         }
 
-        callback(response_status_code, response, response_headers);
+        ledger::UrlResponse response;
```
Comment (tmancey) [inline vendor/bat-native-confirmations/src/bat/confirmations/internal/redeem_token_unittest.cc:312]: UrlResponse

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/logging_util.cc (hunk 20 above)
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/logging_util.cc:78]: done

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/redeem_token_unittest.cc (hunk 21 above)
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/redeem_token_unittest.cc:209]: done

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/redeem_token_unittest.cc (hunk 22 above)
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/redeem_token_unittest.cc:158]: done

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/redeem_token_unittest.cc (hunk 23 above)
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/redeem_token_unittest.cc:261]: done

File: vendor/bat-native-confirmations/src/bat/confirmations/internal/redeem_token_unittest.cc (hunk 24 above)
Comment (NejcZdovc) [reply vendor/bat-native-confirmations/src/bat/confirmations/internal/redeem_token_unittest.cc:312]: done

File: vendor/bat-native-ledger/include/bat/ledger/ledger_client.h (hunk 8 above)
Comment (tmancey) [reply vendor/bat-native-ledger/include/bat/ledger/ledger_client.h:28]: oh yeah lol

**Comment by NejcZdovc** [comment]
windows was aborted, restarting

**Pull request merged**